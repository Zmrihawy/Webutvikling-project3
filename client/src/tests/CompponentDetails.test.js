import React from "react";
import ComponentDetails from "../pages/componentDetails/ComponentDetails";
import renderer from "react-test-renderer";

it("renders shopping cart page", () => {
  const  currentComponent = null;
  const getCurrentComponent = null;
  const addComponentToShoppingCart = null;
  const loggedInUser = null;
  const match= null;
  const tree = renderer.
  create(<shoppingCartPage
    currentComponent = {currentComponent}
    getCurrentComponent = {getCurrentComponent}
    addComponentToShoppingCart = {addComponentToShoppingCart} 
    loggedInUser = {loggedInUser}
    match = {match}/>).toJSON();
  expect(tree).toMatchSnapshot();
});
