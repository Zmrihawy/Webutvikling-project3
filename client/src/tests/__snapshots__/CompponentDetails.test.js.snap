// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders shopping cart page 1`] = `
<shoppingCartPage
  addComponentToShoppingCart={null}
  currentComponent={null}
  getCurrentComponent={null}
  loggedInUser={null}
  match={null}
/>
`;
